### DISCUSSION TITLE
Create Menu always appears and cannot be closed

### DISCUSSION THREAD

**Discussion by curly210102** [community/community#56285] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

When opening any page on the github.com website, the Create Menu always appears and clicking the button does not close it. 
I have tried disabling all browser extensions but the issue persists.

https://github.com/community/community/assets/76864176/dd3c8a74-760c-4fc3-b47f-23ce0db58714

**Comment by bstst** [comment]
Same here! Help!

It's always like this:

<img width="1441" alt="image" src="https://github.com/community/community/assets/412859/9cfd3c83-c4a3-413d-8341-c37ec881ab63">

**Reply by curly210102** [reply, marked as answer]
It's happened with the preview feature Global navigator, https://github.com/orgs/community/discussions/52083#discussioncomment-5983786, I disable the feature temporarily

**Comment by martineausimon** [comment]
Same here, with qutebrowser v3.0.0
